Meadowlark-js/packages/meadowlark-authz-server/src/security/TokenValidator.ts

```diff
@@ -139,7 +139,7 @@ export async function introspectBearerToken(bearerToken: string, traceId: string
   const jwt: Jwt = verified as Jwt;
 
   // Check for correct issuer
-  if (jwt.body?.iss !== TOKEN_ISSUER || jwt.body?.aud !== TOKEN_ISSUER) return { isValid: false };
+  if (jwt.body?.iss !== getTokenIssuer() || jwt.body?.aud !== getTokenAudience()) return { isValid: false };
```

Comment:
Previously, `iss` and `aud` were the same. Since I was in this code, I separated them. This helps get us closer to integrating 3rd party OAuth2.